xrpl/transaction/integration/payment_test.go

```diff
@@ -0,0 +1,124 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/Peersyst/xrpl-go/xrpl/rpc"
+	"github.com/Peersyst/xrpl-go/xrpl/testutil/integration"
+	"github.com/Peersyst/xrpl-go/xrpl/transaction"
+	"github.com/Peersyst/xrpl-go/xrpl/transaction/types"
+	"github.com/Peersyst/xrpl-go/xrpl/websocket"
+	"github.com/stretchr/testify/require"
+)
+
+type PaymentTest struct {
+	Name          string
+	Payment       *transaction.Payment
+	ExpectedError string
+}
+
+func TestIntegrationPayment_Websocket(t *testing.T) {
+	env := integration.GetEnv(t)
+	client := websocket.NewClient(websocket.NewClientConfig().WithHost(env.Host).WithFaucetProvider(env.FaucetProvider))
+
+	runner := integration.NewRunner(t, client, &integration.RunnerConfig{
+		WalletCount: 2,
+	})
+
+	err := runner.Setup()
+	require.NoError(t, err)
+	defer runner.Teardown()
+
+	sender := runner.GetWallet(0)
+	receiver := runner.GetWallet(1)
+
+	tt := []PaymentTest{
+		{
+			Name: "pass - XRP to XRP",
+			Payment: &transaction.Payment{
+				BaseTx: transaction.BaseTx{
+					Account: sender.GetAddress(),
+				},
+				Amount:      types.XRPCurrencyAmount(1),
+				Destination: receiver.GetAddress(),
+			},
+		},
+		{
+			Name: "fail - missing Destination",
+			Payment: &transaction.Payment{
+				BaseTx: transaction.BaseTx{
+					Account: sender.GetAddress(),
+				},
+				Amount: types.XRPCurrencyAmount(1),
+			},
+			ExpectedError: "invalidTransaction",
```

Comment:
Would it make sense to extract those error messages into their own file for better reusability?

https://xrpl.org/docs/references/http-websocket-apis/public-api-methods/transaction-methods/submit#possible-errors